server/tests/routes/user.test.js

```diff
@@ -339,77 +339,4 @@ describe('## User APIs', () => {
         });
     });
   });
-
```

Comment:
This is a lot of tests ripped out with none added. It'd be good to have some tests around group population